src/specializations/Triangle.jl

```diff
@@ -111,16 +111,19 @@ _has_analytical(::Type{T}) where {T <: Meshes.Triangle} = true
 #                              Parametric
 ################################################################################
 
-function _parametric(triangle::Meshes.Triangle, t1, t2)
-    if (t1 < 0 || t1 > 1) || (t2 < 0 || t2 > 1)
-        msg = "triangle(t1, t2) is not defined for (t1, t2) outside [0, 1]^2."
-        throw(DomainError((t1, t2), msg))
+function _parametric(triangle::Meshes.Triangle)
+    function f(t1, t2)
+        if any(Iterators.map(n -> (n < 0) || (n > 1), (t1, t2)))
+            msg = "triangle(t1, t2) is not defined for (t1, t2) outside [0, 1]^2."
```

Comment:
```suggestion
            msg = "triangle(t1, t2) is not defined for (t1, t2) outside [0, 1]²."
```
This just looks a bit nicer (and is how it looks like in Meshes.jl) :smiley:

Reply:
Agreed. I just commit other changes to this file and included the suggested change there.